sdks/python/apache_beam/testing/data/trigger_transcripts.yaml

```diff
@@ -163,6 +169,8 @@ transcript:
 
 ---
 name: known_late_data_no_watermark_hold_possible
+broken_on:
```

Comment:
IIUC, with this change, latter when we add similar TestStream tests, we just change this .yaml file to add new test cases, right?